test/test_geant4.jl

```diff
@@ -66,11 +67,19 @@ end
     @test evts isa Table
     @test length(evts) == 100
 
+    # Temporary HDF5 file
+    mktemp() do tmpfile, io
+        LegendHDF5IO.lh5open(tmpfile, "w") do h5
+            LegendHDF5IO.writedata(h5, "geant4", evts)
+        end
+        @test isfile(tmpfile)
+    end
```

Comment:
I would also be interested if this can be read back in and gives the same table:

```suggestion
    mktemp() do tmpfile, io
        LegendHDF5IO.lh5open(tmpfile, "w") do h5
            LegendHDF5IO.writedata(h5, "geant4", evts)
        end
        @test isfile(tmpfile)
        
        evts == LegendHDF5IO.lh5open(tmpfile) do h5
            LegendHDF5IO.readdata(h5, "geant4")
        end
    end
```